GitUp/Application/AppDelegate.m

```diff
@@ -204,8 +236,6 @@ - (void)_openDocument:(NSMenuItem*)sender {
 
 - (void)awakeFromNib {
 
-  _allowWelcome = -1;
```

Comment:
What is the reason to set `_allowWelcome` to -1? Welcome window lifecycle perfectly works without it... It can be boolean, I guess.

Reply:
It's used to indicate should show (1), explicitly closed (0), and was never activated yet (-1). You can see this if you close the window and then command-tab away and then back to the app (the welcome window won't activate itself).

Reply:
Don't think it can without changing behavior.

Reply:
@lucasderraugh Is it desired behavior? A bit confused, though.

Reply:
@lucasderraugh I restored this behavior.